ivy/functional/ivy/creation.py

```diff
@@ -1186,7 +1198,7 @@ def eye(
                 [0., 1., 0.]]])
     >>> x1.shape
     (3, 2, 3)
-    # Suppose batch_shape = [a, b] then the returning identity array shape is [a, b, numRows, numColumns]
+    # Suppose batch_shape = [a, b] then the returning identity array shape is [a, b, numRows, numColumns] # noqa
```

Comment:
Not really sure of the context of the comment, is it better to be added as a plain text comment instead of being a code comment?
```suggestion

    Suppose batch_shape = [a, b] then the returning identity array shape is [a, b, numRows, numColumns] # noqa
```

Reply:
Yeah, currently it is being rendered as a comment inside the code block.
I believe keeping it as plain text outside would be sufficient.
![Screenshot (1352)](https://github.com/unifyai/ivy/assets/61016383/dbdd9cb6-bc53-453a-8568-f44d0d58238e)